package.json

```
@@ -1,19 +1,19 @@
 {
-  "name": "@digitalbazaar/did-method-key",
+  "name": "@digitalbazaar/did-method-web",
   "version": "3.0.1-0",
```

Reset so it works for initial publishing.
```suggestion
  "version": "0.0.1-0",
```